client/pages/sections/protocols/animals.js

```diff
@@ -46,10 +34,9 @@ function normaliseValues(speciesDetails) {
 
 export function filterSpeciesByActive(protocol, project) {
   const { species = [], speciesDetails = [] } = protocol;
-  const projectSpecies = getProjectSpecies(project);
 
   return normaliseValues(speciesDetails).filter(s => {
-    return (species.includes(s.value) || species.includes(s.name)) && (projectSpecies.includes(s.value) || projectSpecies.includes(s.name));
```

Comment:
We may need to keep this for backwards compatibility with old data where the protocols still have species included in data that are not on the top level project.

They won't be removed until they're edited.